openmls/src/schedule/application_export_tree.rs

```diff
@@ -0,0 +1,29 @@
+use openmls_traits::{crypto::OpenMlsCrypto, types::Ciphersuite};
+
+use crate::{
+    binary_tree::array_representation::TreeSize,
+    ciphersuite::Secret,
+    schedule::{
+        pprf::{Pprf, PprfError, Prefix16},
+        ApplicationExportSecret,
+    },
+};
+
+pub(crate) type ApplicationExportTree = Pprf<Prefix16>;
+pub(crate) type ApplicationExportTreeError = PprfError;
+
+impl ApplicationExportTree {
+    pub fn new(application_exporter: ApplicationExportSecret) -> Self {
```

Comment:
The type isn't `pub`. So making these `pub` doesn't make much sense. Is this supposed to be used on the outside?